codegen/src/api/mod.rs

```diff
@@ -12,20 +12,20 @@ mod events;
 mod runtime_apis;
 mod storage;
 
+use scale_typegen::typegen::ir::type_ir::{CompositeFieldIR, CompositeIR, CompositeIRKind};
```

Comment:
would be nice to export all these types from the root in `scale_typegen` all the types are already unique and I don't think for instance `scale_typegen::typegen::ir::type_ir::CompositeFieldIR` is necessary and `scale_typegen::CompositeFieldIR` would be sufficient.

Nothing to address here just a comment

Reply:
Good point! We already expose some other types at the root, I did not anticipate that we would need easy access to these structs.